src/NumberTheory/GaloisGrp/GaloisGrp.jl

```diff
@@ -2257,6 +2257,7 @@ function relative_invariant(G, U; Chain::Union{Nothing, <:Vector{<:Tuple{PermGro
       if length(d) == length(T)
         break
       end
+      r = [rand(k) for i=1:degree(G)]
```

Comment:
`r` is not used, is it? So all this does is to perturb the current state of the default Julia RNG, right?

Reply:
5 lines above

Reply:
I need to compare sl-polys. In general I cannot, writing them as mpolys is impossible, so I do the next best thing: In the end I need to prove that two polys are different. So I find random evaluation points and compare the evaluation.

If then eval different, then the polys are different. If they eval is equal I did not lear anything.

In here, the eval poitns are chosen once and for all at the top. However, after a comparison returned equal, I am changing them - as my previous batch might have been dud.

Reply:
My bad, I missed the use of `r` a few lines above in line 2256